pilot/docker/Dockerfile.proxyv2

```diff
@@ -11,6 +11,9 @@ FROM docker.io/istio/base:${BASE_VERSION} as default
 # obtained from debian ca-certs deb using fetch_cacerts.sh
 ADD ca-certificates.tgz /
 
+# Add WASM plugins for stats and metadata_exchange
+ADD wasm /
```

Comment:
what is the resulting path in the container?
`/wasm/stats_1_3_1.wasm` ?

Reply:
yup : /wasm/stats_1_3_1.wasm

Reply:
Can you load the plugin into 1.3.1 ?

@jplevyak is this .wasm file compatible with what is in 1.3 branch or master only ?
@kyessenov any breaking changes?

Reply:
Master has breaking changes in envoy-wasm. I am not sure which SHA was used to cut-off 1.3.1.

Reply:
1.3.1 will use the current HEADs of the `release-1.3` branches.